chcemvediet/apps/inforequests/admin.py

```diff
@@ -316,8 +317,8 @@ def get_queryset(self, request):
         queryset = queryset.select_related(u'advanced_by')
         return queryset
 
-    def get_inforequest(self, obj):
-        return obj.inforequest
+    def get_inforequest(self, objs):
+        return Inforequest.objects.filter(branch__in=objs)
```

Comment:
Pri `delete_selected` je `objs` typu `QuerySet`, pri `render_delete_form` to je typu `[int]`  *(list s jednym integerom). Selector `__in` funguje pre oba typy.

Reply:
jj, dokonca je to take inteligentne, ze ak `objs` nie je pole, ale queryset, tak to nespravi dva selecty, ale ten queryset pouzije ako subquery..

Zislo by sa vsak premenovat metodu `get_inforequest` na `get_inforequests` (pridane `s` na konci), kedze to teraz vracia pole, nie jeden prvom. A potom analogicky vsetky premenne po ceste premenovat do mnozneho cisla. Aby bolo jasne, ze je to pole inforequestov, nie jeden inforequest.

Reply:
jj, premenne aj parametre som premenoval do mnozneho cisla, na nazov metdy som zabudol, dakujem, :ok: